client_operations_test.go

```diff
@@ -20,10 +20,7 @@ func TestCreateAccount(test *testing.T) {
 			return mockResponse(req, 201, String(`{"id": "abcd1234"}`))
 		},
 	}
-	client, err := scenario.MockHTTPClient(nil)
```

Comment:
This is an example of the breaking change created by introducing an `error` to the `NewClient` function. These tests should not need to be changed.